pallets/registry/src/tests.rs

```diff
@@ -0,0 +1,294 @@
+	let author = ACCOUNT_00;
+	let raw_registry = [2u8; 256].to_vec();
+	let registry: InputRegistryOf<Test> = BoundedVec::try_from(raw_registry).unwrap();
+	let id_digest = <Test as frame_system::Config>::Hashing::hash(
+		&[&registry.encode()[..], &creator.encode()[..]].concat()[..],
+	);
+	let registry_id: RegistryIdOf = generate_registry_id::<Test>(&id_digest);
+	new_test_ext().execute_with(|| {
+		//creating regisrty
+		assert_ok!(Registry::create(
+			DoubleOrigin(author.clone(), creator.clone()).into(),
+			registry.clone(),
+			None
+		));
+
+		//Checking whether registry creator and creator are different
+		assert_ne!(<Registries<Test>>::get(&registry_id).unwrap().creator, DID_01);
+
+		assert_err!(
+			Registry::is_an_authority(&registry_id, DID_01),
+			Error::<Test>::UnauthorizedOperation
+		);
+	});
+}
+
+#[test]
+fn add_admin_delegate_should_fail_if_delegate_is_already_added() {
+	env_logger::init();
+	let creator = DID_00;
+	let author = ACCOUNT_00;
+	let raw_registry = [2u8; 256].to_vec();
+	let registry: InputRegistryOf<Test> = BoundedVec::try_from(raw_registry).unwrap();
+	let id_digest = <Test as frame_system::Config>::Hashing::hash(
+		&[&registry.encode()[..], &creator.encode()[..]].concat()[..],
+	);
+	let registry_id: RegistryIdOf = generate_registry_id::<Test>(&id_digest);
+	// let authorization_id: Ss58Identifier =
+	// 	Ss58Identifier::to_authorization_id(&id_digest.encode()[..]).unwrap();
+
+	//Schema creation from schema pallet
+	let raw_schema = [2u8; 256].to_vec();
+	let schema: InputSchemaOf<Test> = BoundedVec::try_from(raw_schema)
+		.expect("Test Schema should fit into the expected input length of for the test runtime.");
+	let schema_id_digest = <Test as frame_system::Config>::Hashing::hash(
+		&[&schema.encode()[..], &creator.encode()[..]].concat()[..],
+	);
+	// let schema_digest: SchemaHashOf<Test> =
+	// 	<Test as frame_system::Config>::Hashing::hash(&schema[..]);
+	let schema_id_of: Ss58Identifier =
+		Ss58Identifier::to_schema_id(&schema_id_digest.encode()[..]).unwrap();
+
+	let new_block_number: BlockNumber = 1;
+
+	new_test_ext().execute_with(|| {
+		//adding schema
+		System::set_block_number(new_block_number);
+		//creating regisrty
+		assert_ok!(Schema::create(
+			DoubleOrigin(author.clone(), creator.clone()).into(),
+			schema.clone()
+		));
+
+		//creating regisrty
+		assert_ok!(Registry::create(
+			DoubleOrigin(author.clone(), creator.clone()).into(),
+			registry.clone(),
+			Some(schema_id_of)
+		));
+
+		//Admin should be able to add the delegate
+		assert_ok!(Registry::add_admin_delegate(
+			DoubleOrigin(author.clone(), creator.clone()).into(),
+			registry_id.clone(),
+			DID_00,
+		));
+
+		//When Trying to add same delegate again it says delegate already added
+		assert_err!(
+			Registry::add_admin_delegate(
+				DoubleOrigin(author.clone(), DID_00.clone()).into(),
+				registry_id.clone(),
+				DID_00,
+			),
+			Error::<Test>::DelegateAlreadyAdded
+		);
+	});
+}
+
+#[test]
+
```

Comment:
I think there shouldn't be a new line between the attribute macro and function

Reply:
Ya, this must be a typo will fix in next commit. Thanks